rclcpp/include/rclcpp/publisher.hpp

```diff
@@ -116,6 +121,10 @@ class PublisherBase
   const rcl_publisher_t *
   get_publisher_handle() const;
 
+  RCLCPP_PUBLIC
```

Comment:
Please add documentation to new public functions (I know we have a lot of catching up to do, but still new functions we should try to add docs from day 1).

Reply:
Done.